config/routes.rb

```diff
@@ -1,3 +1,12 @@
 Rails.application.routes.draw do
+  root 'inventories#index'
+  devise_for :accounts
+  resources :inventories
+  resources :brands, only: [:index, :destroy]
+  resources :models, only: [:destroy]
+  post 'brands/create'
+  patch 'brands/update'
+  post 'models/create'
+  patch 'models/update'
```

Comment:
Trong rails rất tuân thủ nghiêm ngặt về Restful. https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions
For example:  PATCH /brands/:id

Do em thiết kế gộp màn hình nên thành ra mới phải tạo những routes "ngoại lệ"
Nếu có nhiều routes như thế này thì routes  file rất dễ bị phình và khó kiểm soát.
Lần sau lưu ý vấn đề này nha. Mỗi màn hình nên xử lý 1 vấn đề, đừng nhét vào hết.
Dễ cho mình và cho ai đó nếu họ join vào làm chung.

Reply:
Okk I understand